VRDR/IJEMortality.cs

```diff
@@ -4411,29 +4421,49 @@ public string DMIDDLE
         [IJEField(167, 1858, 50, "Father's First Name", "DDADF", 1)]
         public string DDADF
         {
+            // TODO: Implement mapping from FHIR record location: DecedentFather
             get
             {
-                // TODO: Implement mapping from FHIR record location: DecedentFather
+                string[] names = record.FatherGivenNames;
+                if (names != null && names.Length > 0)
+                {
+                    return names[0];
+                }
                 return "";
```

Comment:
needs left justified

Reply:
This is handled the same as GNAME and MNAME. So either we change all of them or none of them.